src/main/resources/GoogleApps/Groovy.groovy

```diff
@@ -352,7 +355,7 @@ public class GoogleAppsGroovy {
                     String emailA = address.getValue();
                     if(emailA.endsWith(DOMAIN)) {
                         foundCompatibleDomain = true;
-                        email = emailA;
+                        usersEmailAddress = emailA;
```

Comment:
I think @mflorea was talking about replacing `emailA` and not `email`, in this case.

Reply:
Yes, I saw that and changed it later on.